usagov/usagov-2021
# Usagov 1860 display usps error

## Jira Task


https://cm-jira.usa.gov/browse/USAGOV-1860

## Description

Update code to get USPS error text to display in the form.
     Changed the reference to the correct password variable
     Changed what is returned in the try fetch
## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions



### Change Requirements



### Validation Steps
Goto http://localhost/elected-officials
Fill out the form with zip field with "1234" and you should get "Please enter a valid 5-digit ZIP code."

Fill out the zip with "12345" and the street address with "El Ray Lane" city with "Sonora" and state with "California" when you enter you should see the street field display an error message "Address not found. Please enter a valid address."

Fill out the zip with "12345" and the street address with "El Ray Lane" city with "Ssss" and state with "California" when you enter you should see the street field display an error message "City not found. Please enter a valid city."

<img width="1512" alt="Screenshot 2024-09-12 at 10 15 06 AM" src="https://github.com/user-attachments/assets/d0b40f6d-6ff3-4b9b-81d8-51ca3ba91baf">

<img width="1512" alt="Screenshot 2024-09-12 at 10 07 42 AM" src="https://github.com/user-attachments/assets/299f9113-6a5e-49d1-9650-7d747feb9550">

<img width="1512" alt="Screenshot 2024-09-12 at 10 11 36 AM" src="https://github.com/user-attachments/assets/9bbe2062-93eb-435f-a270-53e6dca05211">

## Security Review




## Reviewer Reminders

- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions

Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
